test/bind.spec.ts

```diff
@@ -44,15 +44,15 @@ describe("live properties", () => {
 
     const size1 = p.get("size")
     const opacity1 = p.get("opacity")
-    expect(Math.abs(size1 - 20.5)).toBeLessThan(0.12)
-    expect(Math.abs(opacity1 - 0.5)).toBeLessThan(0.12)
+    expect(Math.abs(size1 - 20.5)).toBeLessThan(0.2)
+    expect(Math.abs(opacity1 - 0.5)).toBeLessThan(0.2)
```

Comment:
This is unrelated to the other changes in this PR... These tests were added in #45 ... I do think they are worthwhile, but I was a little worried they'd be flaky, and they seem to be. My goal was to test that the live props actually update. I'm not too concerned with asserting that after 500ms, the `t` value has advanced by exactly 500. I'm not really sure, but I suspect this is implemented via Threestrap, and I know threestrap has more fine-grained tests like that.

Reply:
yeah, checking `t` like that would be silly unless we can inject our own stopwatch for testing. Fine by me!